src/utils/retry.ts

```diff
@@ -0,0 +1,24 @@
+import { sleep } from '../integration/testHelpers';
+
+export const retry = async (
```

Comment:
Does this actually need to move to the `/utils/` folder rather than remain in `testHelpers.ts`. This function was written explicitly for retrying flakey integration tests, so it feels like it makes more sense there at the moment.

This folder doesn't expose anything to users, nothing in here is exported out.